composer.json

```
@@ -53,6 +53,8 @@
         "symfony/twig-bundle": "^4.3",
         "symfony/validator": "^4.3",
         "twig/extensions": "^1.5",
+        "twig/extra-bundle": "^3.0",
+        "twig/string-extra": "^3.0",
         "twig/twig": "^2.10"
```

```suggestion
        "twig/twig": "^2.12.1"
```
Can we bump here twig? so is [when it was added the String extension](https://github.com/twigphp/Twig/blob/2.x/CHANGELOG) and also get rid of some deprecations with php 7.4

---

Done.